### DISCUSSION TITLE
live_transcribe？

### DISCUSSION THREAD

**Discussion by IIZYFII** [Purfview/whisper-standalone-win#27] [Ideas]
Dear developer, hello

How can transcription function be achieved?

I couldn't find response support on the command line

Do you have any plans to add this feature

**Comment by Purfview** [comment]
No plans for such feature.

**Comment by Purfview** [comment]
UPDATE:
I looked at jordimas' implementation of live transcribe, it doesn't work properly for me, it mostly outputs some nonsense, maybe it's some bug with Windows or whatever, I don't have time to debug it.
So, still no plans.

**Comment by patrickmac110** [comment]
What about implementing this? [https://github.com/ufal/whisper_streaming](https://github.com/ufal/whisper_streaming)
I'm no developer, and I have been struggling to get the dependencies installed, but just thought I'd bring it up...
